src/Models/Traits/HasUsageReceiptItems.php

```
@@ -16,20 +16,21 @@ trait HasUsageReceiptItems
      */
     protected Collection $items;
 
-    protected ?float $forcedTotal = null;
+    protected ?float $expectedTotal = null;
+
+    protected bool $throwOnExpectedTotalMismatch = false;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $throwOnExpectedTotalMismatch. Keep variable name length under 20. (undefined)

(LongVariable)